sample_size/scripts/utils.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict
+from typing import Tuple
+
+from sample_size.sample_size_calculator import DEFAULT_ALPHA
+
+METRIC_PARAMETERS = {
+    "boolean": {"probability": "baseline probability (between 0 and 1)"},
+    "numeric": {"variance": "variance of the baseline metric"},
+    "ratio": {
+        "numerator_mean": "mean of the baseline metric's numerator",
+        "numerator_variance": "variance of the baseline metric's numerator",
+        "denominator_mean": "mean of the baseline metric's denominator",
+        "denominator_variance": "variance of the baseline metric's denominator",
+        "covariance": "covariance between the baseline metric's numerator and denominator",
+    },
+}
+
+
+def is_float(value: str) -> float:
```

Comment:
this is returning bool right? Does mypy pass this check?

Reply:
yeah somehow it passed. will update